### PR TITLE
trim immense terms so they don't cause graylog to choke

### PR DESCRIPTION
**Pull request by service-michaelnielson-taulia**

### CODE REVIEW

**Comment by service-michaelnielson-taulia** [comment]
@trailrunner-taulia I think this is a good thing to do.

See this fun: http://qetest-graylog2.taulia.com/system/indices/failures

**Comment by service-michaelnielson-taulia** [comment]
http://qetest-graylog2.taulia.com/messages/graylog2_208/ef5bc650-655a-11e6-af89-0ac8156b3ed4#fields=message%2Csource

**Comment by trailrunner-taulia** [comment]
👍  Nice! Shall I merge?

File: src/test/java/org/graylog2/log4j2/GelfAppenderTest.java [hunk 1]
```diff
@@ -49,6 +49,20 @@ public void testThreadContext() {
...
+    public void testImmenseStackTrace() throws Exception {
+        final Logger logger = LogManager.getLogger("test");
+        logger.info("This message is huge!", makeHugeMessWithSuperLongMethodNameJustSoWeAreSureToBreakThe32kLimitAmIRight());
+    }
+
+    private static Exception makeHugeMessWithSuperLongMethodNameJustSoWeAreSureToBreakThe32kLimitAmIRight() {
+        try {
+            return makeHugeMessWithSuperLongMethodNameJustSoWeAreSureToBreakThe32kLimitAmIRight();
+        } catch (StackOverflowError e) {
+            return new Exception("really long exception message", e);
```
Comment (trailrunner-taulia) [inline src/test/java/org/graylog2/log4j2/GelfAppenderTest.java:62]: Creative!

Comment (service-michaelnielson-taulia) [inline src/test/java/org/graylog2/log4j2/GelfAppenderTest.java:62]: A really long message would have been fine too, but this was more fun.

**Comment by service-michaelnielson-taulia** [comment]
👍 yes please merge @trailrunner-taulia

**Pull request merged**